src/herbpy/barretthand.py

```diff
@@ -109,7 +109,7 @@ def GetIndices(self):
         order: [ finger 0, finger 1, finger 2, spread ].
         @return DOF indices of the hand
         """
-        return [ self.GetSpreadIndex() ] + self.GetFingerIndices()
+        return [self.GetSpreadIndex()] + self.GetFingerIndices()
```

Comment:
This does not match the docstring, which explicitly specifies the order ` [ finger 0, finger 1, finger 2, spread ]`.